Packs/Gmail/Integrations/Gmail/Gmail.py

```diff
@@ -2129,62 +2129,58 @@ def send_mail(emailto, emailfrom, subject, body, entry_ids, cc, bcc, htmlBody, r
 
 def send_mail_command():
     args = demisto.args()
-    emailto = argToList(args.get('to'))
-    emailfrom = args.get('from')
+    return mail_command(args)
+
+
+def mail_command(args, subject_prefix='', in_reply_to=None, references=None):
+    email_to = argToList(args.get('to'))
+    email_from = args.get('from')
     body = args.get('body')
-    subject = args.get('subject')
+    subject = f"{subject_prefix}{args.get('subject')}"
     entry_ids = argToList(args.get('attachIDs'))
     cc = argToList(args.get('cc'))
     bcc = argToList(args.get('bcc'))
-    htmlBody = args.get('htmlBody')
-    replyTo = args.get('replyTo')
+    html_body = args.get('htmlBody')
+    reply_to = args.get('replyTo')
     file_names = argToList(args.get('attachNames'))
```

Comment:
While we're aligning variable names, what do you think about renaming this one to `attach_names`?

Reply:
done